code/API_definitions/qos-booking-and-assignment.yaml

```
@@ -1155,7 +1155,7 @@ components:
     PhoneNumber:
       description: A public identifier addressing a telephone subscription. In mobile networks it corresponds to the MSISDN (Mobile Station International Subscriber Directory Number). In order to be globally unique it has to be formatted in international format, according to E.164 standard, prefixed with '+'.
       type: string
-      pattern: '^\+[1-9][0-9]{4,14}$'
+      pattern: "^\+[1-9][0-9]{4,14}$"

```

The pattern: `'^\+[1-9][0-9]{4,14}$'` is single quoted in Commonalities to keep it literally, and must not be double quoted (see the syntax error by YAML lint). Please keep it like in Commonalities:

```suggestion
      pattern: '^\+[1-9][0-9]{4,14}$'
```